README.rst

```diff
@@ -680,6 +680,20 @@ executed.
       for task in tiger.current_tasks:
           print(task.n_executions())
 
+Example 4: Preventing processing of an organization's queued tasks by setting them as errored.
```

Comment:
This wording could be more generic. "Drain a specific queue by setting tasks as errored".